tests/app/dao/test_ft_billing_dao.py

```diff
@@ -609,7 +609,7 @@ def test_fetch_sms_billing_for_all_services_without_an_organisation_appears(noti
     setup = set_up_usage_data(datetime(2019, 5, 1))
     results = fetch_sms_billing_for_all_services(datetime(2019, 5, 1), datetime(2019, 5, 31))
 
-    assert len(results) == 2
+    assert len(results) == 3
```

Comment:
How come this changed here (in the last commit)?

Reply:
It's because I added extra item to the fixture (service with sms within allowance)

Reply:
🙈 don't mind me - I think I must have misread something!

Reply:
Ah wait, I did read it right: I think this should be in the _previous_ commit "Billing report only has services with bills to pay". It confused me when I first read it, but I'm used to it now, so I think it's OK to leave it as-is.